test/version/version.test.js

```diff
@@ -1,374 +1,296 @@
         expect(stderr).toBeFalsy();
-        expect(stdout).toContain(`@webpack-cli/generators ${generatorsPkgJSON.version}`);
-        expect(stdout).toContain(`webpack-cli ${pkgJSON.version}`);
-        expect(stdout).toContain(`webpack ${webpack.version}`);
-        expect(stdout).toContain(`webpack-dev-server ${webpackDevServerPkgJSON.version}`);
+        expect(serializeSnapshot(stdout)).toMatchSnapshot();
     });
 
     it('outputs version with loader', () => {
         const { exitCode, stderr, stdout } = run(__dirname, ['loader', '--version']);
 
         expect(exitCode).toBe(0);
         expect(stderr).toBeFalsy();
-        expect(stdout).toContain(`@webpack-cli/generators ${generatorsPkgJSON.version}`);
-        expect(stdout).toContain(`webpack-cli ${pkgJSON.version}`);
-        expect(stdout).toContain(`webpack ${webpack.version}`);
-        expect(stdout).toContain(`webpack-dev-server ${webpackDevServerPkgJSON.version}`);
+        expect(serializeSnapshot(stdout)).toMatchSnapshot();
     });
 
     it('outputs version with init', () => {
         const { exitCode, stderr, stdout } = run(__dirname, ['init', '--version']);
 
         expect(exitCode).toBe(0);
         expect(stderr).toBeFalsy();
-        expect(stdout).toContain(`@webpack-cli/generators ${generatorsPkgJSON.version}`);
-        expect(stdout).toContain(`webpack-cli ${pkgJSON.version}`);
-        expect(stdout).toContain(`webpack ${webpack.version}`);
-        expect(stdout).toContain(`webpack-dev-server ${webpackDevServerPkgJSON.version}`);
+        expect(serializeSnapshot(stdout)).toMatchSnapshot();
     });
 
     it('outputs version with serve', () => {
         const { exitCode, stderr, stdout } = run(__dirname, ['serve', '--version']);
 
         expect(exitCode).toBe(0);
         expect(stderr).toBeFalsy();
-        expect(stdout).toContain(`@webpack-cli/serve ${servePkgJSON.version}`);
-        expect(stdout).toContain(`webpack-cli ${pkgJSON.version}`);
-        expect(stdout).toContain(`webpack ${webpack.version}`);
-        expect(stdout).toContain(`webpack-dev-server ${webpackDevServerPkgJSON.version}`);
+        expect(serializeSnapshot(stdout)).toMatchSnapshot();
     });
 
     it('outputs version with migrate', () => {
         const { exitCode, stderr, stdout } = run(__dirname, ['migrate', '--version']);
         expect(exitCode).toBe(0);
         expect(stderr).toBeFalsy();
-        expect(stdout).toContain(`webpack-cli ${pkgJSON.version}`);
-        expect(stdout).toContain(`@webpack-cli/migrate`);
-        expect(stdout).toContain(`webpack ${webpack.version}`);
-        expect(stdout).toContain(`webpack-dev-server ${webpackDevServerPkgJSON.version}`);
+        expect(serializeSnapshot(stdout)).toMatchSnapshot();
     });
 
     it('outputs version with the alias c for init', () => {
         const { exitCode, stderr, stdout } = run(__dirname, ['c', '--version']);
 
         expect(exitCode).toBe(0);
         expect(stderr).toBeFalsy();
```

Comment:
Ideally we should have `toMatchSnapshot` here too, yes it will be empty in shapshot, but when you debug you will see which errors happens and output, easy to debug

Reply:
yep makes sense

Reply:
done